### PR TITLE
Skip bumping EKB images

### PR DESCRIPTION
**Pull request by upodroid**
@Cali0707 @matzew 

This might work

### CODE REVIEW

**Review by Cali0707 — APPROVED** [review]
These changes seem to make sense, and the images referenced are now pre-JDK21 for EKB (yay!)

/lgtm
/hold in case @matzew wants to take a look as well

File: prow/jobs_config/knative-extensions/eventing-kafka-broker.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 org: knative-extensions
 repo: eventing-kafka-broker
 branches: [main]
-image: us-docker.pkg.dev/knative-tests/images/prow-tests:v20231207-30ee442a2
+image: us-docker.pkg.dev/knative-tests/images/prow-tests:v20231129-81431e1fe
```
Comment (creydr) [inline prow/jobs_config/knative-extensions/eventing-kafka-broker.yaml:4]: nit: Even this change will probably be overridden by the next image bumper job, we could stay at `v20231207-30ee442a2` for main already

**Comment by pierDipi** [comment]
This looks good

@upodroid can we unhold ?

**Comment by upodroid** [comment]
This didn't work

holding this PR that is trying to bump the images

https://github.com/knative/infra/pull/308

**Comment by Cali0707** [comment]
@upodroid it looks like this PR also rolled back the main branch to JDK21

**Comment by pierDipi** [comment]
I think there is no support for regex for `excludedConfigPaths`

- https://github.com/kubernetes/test-infra/blob/5005fda29ff65601f4f6e6c71b560bf2992bca80/prow/cmd/generic-autobumper/main.go#L360C43-L360C54
- https://github.com/kubernetes/test-infra/blob/5005fda29ff65601f4f6e6c71b560bf2992bca80/prow/cmd/generic-autobumper/helper.go#L71-L78

**Comment by pierDipi** [comment]
See https://github.com/knative/infra/pull/309

**Pull request merged**